Added french translation

inthavoc opened the pull request:
French translation for some terms in the UI

inthavoc committed:
french locale

GrapheneCt commented:
psvs_msg_hud_off, psvs_msg_hud_full and psvs_msg_hud_pos are too long and will not display correctly.

inthavoc committed:
Update psvshell_locale_fr.xml

psvs_msg_hud_pos, psvs_msg_hud_full and psvs_msg_hud_off were abbreviated to meet string length requirements.

inthavoc commented:
Corrected, shouldn't be an issue anymore.
Thanks
